app/components/FlowNavigation.tsx

```diff
@@ -44,45 +37,43 @@ const stateIsDone = (state: NavState) =>
 const stateIsActive = (state: NavState) =>
   [NavState.Current, NavState.Open, NavState.Done].includes(state);
 
-const navItemClassnames = (
-  isCurrent: boolean,
-  isDisabled: boolean,
-  isExpanded = false,
-) =>
-  "p-16 flex gap-x-16 items-center " +
-  (isCurrent ? "ds-label-02-bold " : "ds-label-02-reg hover:underline ") +
-  (isDisabled
-    ? "text-gray-600 curser-not-allowed hover:font-normal pointer-events-none "
-    : " ") +
-  (!isDisabled && isExpanded ? "border-b-2 border-white" : "");
-
 type NavigationA11yLabels = {
   menuLabel: string;
   itemFinished: string;
   itemOpen: string;
 };
+type FlowNavigationProps = Readonly<{
+  navItems: NavItem[];
+  a11yLabels?: NavigationA11yLabels;
+}>;
 
-export default function FlowNavigation({
+const NavigationList = ({
   navItems,
   a11yLabels,
-}: Readonly<{
-  navItems: NavItem[];
-  a11yLabels?: NavigationA11yLabels;
-}>) {
+  isChild = false,
+}: FlowNavigationProps & { isChild?: boolean }) => (
+  <ul
+    className={
+      isChild
+        ? "pt-8 pl-32 mr-8 min-w-fit max-w-fit md:min-w-[250px] md:max-w-[250px] break-words"
+        : ""
```

Comment:
maybe we could utilize classNames here

```suggestion
    className={
      classNames({ "pt-8 pl-32 mr-8 min-w-fit max-w-fit md:min-w-[250px] md:max-w-[250px] break-words": isChild })
```

Reply:
Already done in https://github.com/digitalservicebund/a2j-rechtsantragstelle/pull/856/ :)